locking/locks.go

```diff
@@ -318,33 +324,26 @@ func (c *Client) lockIdFromPath(path string) (string, error) {
 	}
 }
 
-// Fetch locked files for the current committer and cache them locally
+// Fetch locked files for the current user and cache them locally
 // This can be used to sync up locked files when moving machines
 func (c *Client) refreshLockCache() error {
-	// TODO: filters don't seem to currently define how to search for a
-	// committer's email. Is it "committer.email"? For now, just iterate
-	locks, err := c.SearchLocks(nil, 0, false)
+	ourLocks, _, err := c.VerifiableLocks(0)
 	if err != nil {
 		return err
 	}
 
 	// We're going to overwrite the entire local cache
 	c.cache.Clear()
-
-	_, email := c.client.CurrentUser()
-	for _, l := range locks {
-		if l.Committer.Email == email {
-			c.cache.Add(l)
-		}
```

Comment:
I think it's still good to have an endpoint that just lists all locks in a flat list, regardless of ownership - it's a fairly fundamental building block and having to combine 2 lists if you're doing something that requires them all would be a pain. It's also more intuitive for `locks` to just list all locks I think.

I agree with using `VerifiableLocks` here though.